Products/ZenUI3/browser/resources/css/zen-cse.css

```diff
@@ -199,6 +199,10 @@ darker input fields:           #505050
         width: 14px;
     }
 
+    /* ZEN-30116 background changes for panel header */
+    .z-cse .x-panel-header-default {
```

Comment:
**x-panel-header-default** is used in many places.  if we are targeting a specific header, then we should be more specific in targeting it.

Reply:
i suppose that all panels in application should have same header styles that are described in some css rules, and if we want different styles in header in some panel we should write or specific css or add **style** property to panel header